setup.py

```diff
@@ -24,17 +24,17 @@
 _requires = [
     'cryptography>=1.1',
     'pyasn1',
-    'six>=1.9.0',
-    'singledispatch',
+    'six>=1.9.0'
```

Comment:
i think the style here was to leave trailing commas, this could be a slightly tighter diff if it didn't modify the `six>=1.9.0` line.

Reply:
added the trailing comma now.